app/marketing/validators.py

```diff
@@ -8,6 +8,8 @@ def validate_not_burner_domain(email: str):
     """Validate that the provided email is not considered a "burner"
     domain, e.g. someone not interested in committing and contributing to
     the community or the ability to recover their account."""
+    if '@' not in email:
+        return
```

Comment:
Raising a `ValidationError` is probably better here.

Reply:
I thought about that, but I thought that since the default email field throws an 'Invalid email' validation error already we wouldn't need to raise another one. I can do it if you think multiple raises are good. Just my two cents.